include/RAJA/policy/cuda/atomic.hpp

```diff
@@ -590,177 +602,162 @@ struct cuda_atomic {
  * These are atomic in cuda device code and non-atomic otherwise
  */
 RAJA_SUPPRESS_HD_WARN
-template <typename T>
-RAJA_INLINE RAJA_HOST_DEVICE T atomicAdd(cuda_atomic, T volatile *acc, T value)
+template <typename T, typename host_policy>
```

Comment:
Do we want to pick a default policy type for host, or does that not make sense?

Reply:
The host policy comes in as part of the gpu atomic policy. The gpu atomic policies have a defaulted version that uses loop atomic and an explicit version that must be specified. This preserves the behavior of the previous policies like cuda_atomic which is now just an alias for cuda_atomic_explicit<loop_atomic>.

Reply:
Sounds good. Thanks for the explanation.